sdcm/nemesis.py

```diff
@@ -4567,6 +4597,15 @@ def disrupt(self):
         self.disrupt_delete_10_full_partitions()
 
 
+class DeleteLargePartitionWithTimestampMonkey(Nemesis):
```

Comment:
The `large` part is not guaranteed here.

Reply:
Good point, will add a check to see that we have at least 50 rows maybe?

Reply:
I think the LargePartition in the name here only indicate the test case this nemesis only relevant to specific test case
i.e. this whole Nemesis is tied up to a very specific  test configuration

we don't have a clear way to mark those kind of nemesis, but I would recommend checking the test configuration for the expect configuration, i.e. if scylla-bench is used, and it's parameters match the expectation (or at least the config file name that is used)

as for the size, I think `space_node_threshold` to enough size should be o.k. to verify enough data is written.

Reply:
Not needed anymore.